app/components/learnergroup-cohort-user-manager.js

```diff
@@ -24,8 +24,7 @@ export default class LearnergroupCohortUserManagerComponent extends Component {
     }
 
     return this.args.users.filter((user) => {
-      return user.firstName.toLowerCase().includes(filter) ||
-        user.lastName.toLowerCase().includes(filter) ||
+      return user.fullName.toLowerCase().includes(filter) ||
```

Comment:
I know it might be confusing to see results on the screen that don't match the filter you entered, but I think we should continue to filter on first / last in addition to full name. This ensures that a match will show up if someone has changed their name and the admin doesn't know or has forgotten.

Reply:
updated to consider first/last names in filter again.